operations/resiliency/Resiliency_Testing_Procedure.md

```diff
@@ -1,240 +1,386 @@
+      
+      ```bash
+      ncn# kubectl label node ncn-w001 uas-
+      ncn# kubectl label node ncn-w003 uas-
+      ```
+
+1. Verify that WLM is configured with the appropriate workload manager within the created UAI.
    
-   ```bash
-   ncn# kubectl label node ncn-w001 uas-
-   ncn# kubectl label node ncn-w003 uas-
-   ```
-* Within the created UAI, verify that WLM is configured with the appropriate workload manager.
-  Verify the connection string of the created UAI:
+   1. Verify the connection string of the created UAI.
    
-   ```bash
-   ncn# cray uas list
-   ```
+      ```bash
+      ncn# cray uas list
+      ```
 
-   Example output:
+      Example output:
 
-   ```
-   [[results]]
-   username =  "uastest"
-   uai_host =  "ncn-w001"
-   uai_status =  "Running: Ready"
-   uai_connect_string = "ssh uastest@172.30.48.49 -p 31137 -i ~/.ssh/id_rsa"
-   uai_img =  "bis.local:5000/cray/cray-uas-sles15sp1-slurm:latest"
-   uai_age =  "1m"
-   uai_name =  "uai-uastest-5653e9b9"
-   ```
-   Login to the created UAI (example):
+      ```
+      [[results]]
+      username =  "uastest"
+      uai_host =  "ncn-w001"
+      uai_status =  "Running: Ready"
+      uai_connect_string = "ssh uastest@172.30.48.49 -p 31137 -i ~/.ssh/id_rsa"
+      uai_img =  "bis.local:5000/cray/cray-uas-sles15sp1-slurm:latest"
+      uai_age =  "1m"
+      uai_name =  "uai-uastest-5653e9b9"
+      ```
    
-   ```bash
-   ncn# ssh uastest@172.30.48.49 -p 31137 -i ~/.ssh/id_rsa
-   ```
-   To verify the configuration of slurm, for example, within the UAI:
+   1. Log in to the created UAI.
+
+      For example:
+
+      ```bash
+      ncn# ssh uastest@172.30.48.49 -p 31137 -i ~/.ssh/id_rsa
+      ```
    
+   1. Verify the configuration of Slurm, for example, within the UAI:
+   
+      ```bash
+      uastest@uai-uastest-5653e9b9:/lus/uastest> srun -N 4 hostname | sort
+      ```
+
+      Example output:
+
+      ```
+      nid000001
+      nid000002
+      nid000003
+      nid000004
+      ```
+
+1. Copy an MPI application source and WLM batch job files to the UAI.
+
+1. Compile an MPI application with the UAI. Launch application as batch job (not interactive) on compute node(s) that have not been designated, already, for reboots once an NCN is shut down.
+   
+   1. Verify that batch job is running and that application output is streaming to a file. Streaming output will be used to verify that the batch job is still running during resiliency testing. A batch job, when submitted, will designate a log file location. This log file can be accessed to be able to verify that the batch job is continuing to run after an NCN is brought down and once it is back online. Additionally, the `squeue` command can be used to verify that the job continues to run (for Slurm).
+   
+   1. Delete the UAI session when all of the testing is complete.
+   
+      ```bash
+      ncn# cray uas delete --uai-list uai-uastest-5653e9b9
```

Comment:
Replace "ncn# cray uas delete --uai-list uai-uastest-5653e9b9"
with, to match the above UAI:

ncn# cray uas delete --uai-list uai-vers-f8fa541f

Reply:
fixed